.github/workflows/ci.yml

```diff
@@ -0,0 +1,107 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🛑 Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: ⬇️ Checkout
+        uses: actions/checkout@v2
+
+      - name: ⎔ Setup node for problem matcher
+        if: github.event.pull_request
+        uses: actions/setup-node@v2
+
+      - name: 📥 Download deps
+        run: npm install
+
+      - name: 🧪 Run lint
+        run: npm run lint
+
+  build:
+    name: Build
+    runs-on: ${{ matrix.os }}
+    needs: lint
+    strategy:
+      fail-fast: false
+      matrix:
+        node:
+          - 14.17.0
+          - 16
+          - 17
+          - 18
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        eslint:
+          - 7
+          - 8
+
+    steps:
+      - name: 🛑 Cancel Previous Runs
```

Comment:
Can we remove this?

Even if there is a previous build, I would still expect them to be released separately.